app/controllers/users_controller.rb

```diff
@@ -3,6 +3,7 @@ class UsersController < ApplicationController
 
   def index
     @users = User.all
+    @friendships = Friendship.all
```

Comment:
You are not using the `@friendships` variable in your index view. Please delete it. Remember to keep the data safe.